cetmix_tower_server/tests/common.py

```diff
@@ -15,6 +15,10 @@ def setUp(self, *args, **kwargs):
         # Cetmix Tower helper model
         self.CetmixTower = self.env["cetmix.tower"]
 
+        # Tags
+        self.tag_staging = self.env["cx.tower.tag"].create({"name": "Staging"})
```

Comment:
We should be careful with names because references are created automatically from name. 
There is a tag called "Staging" in the [demo data](https://github.com/cetmix/cetmix-tower/blob/a44e50b7b0f0614d2f89ecff514f60246ad9ae74/cetmix_tower_server/demo/demo_data.xml#L18).
So better rename tag in tests, eg `Test Staging` to avoid possible confusions and misunderstanding.
NB: this is valid for all tag occurrences.